Packages.props

```diff
@@ -70,7 +70,7 @@
 		<PackageReference Update="Microsoft.Data.SqlClient" Version="5.1.4" />
 		<PackageReference Update="Microsoft.SqlServer.SqlManagementObjects" Version="170.18.0" />
 		<PackageReference Update="Newtonsoft.Json" Version="13.0.3" />
-		<PackageReference Update="System.Configuration.ConfigurationManager" Version="7.0.0" />
+		<PackageReference Update="System.Configuration.ConfigurationManager" Version="8.0.0" />
```

Comment:
Please also consider upgrading System.IO.Packaging, System.Composition, System.Security.Permissions, System.Text.Encoding.CodePages, System.Text.Encodings.Web, Microsoft.Extensions.DependencyModel and Microsoft.Extensions.FileSystemGlobbing to v8.0.0. Maybe combine them together with a comment to update on every .NET major version bump.

Reply:
Why do those packages also need to be updated?

Reply:
"Need" is a strong word, but in general keeping them updated is a good thing. And major version bumps are a safe time to do that.

Reply:
Because they are built from dotnet/runtime and when it ships v8.0.0, also releases the libraries v8.0.0, so they're best compatible when versions match.